hardhat.config.ts

```diff
@@ -10,6 +10,12 @@ import 'hardhat-abi-exporter';
 import '@nomicfoundation/hardhat-chai-matchers'; /** NEW FEATURE - https://hardhat.org/hardhat-chai-matchers/docs/reference#.revertedwithcustomerror */
 import '@nomicfoundation/hardhat-toolbox'; /** NEW FEATURE */
 import '@openzeppelin/hardhat-upgrades';
+import '@nomiclabs/hardhat-ethers';
```

Comment:
can be removed -> https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-toolbox